### ISSUE TITLE
Word suggestion disappears after completing the word

### ISSUE THREAD

**Issue by jwsp1** [dslul/openboard#11]
When typing a word completely, it disappears on the suggest panel.
E.g. write 'happiness' and watch how it gets suggested after your wirte 'happi' and then gets removed after completing the word 'happiness'.
I know that is it not 100% necessary to show a word that has been written correctly, but other keyboards work that way (e.g. Swiftkey) and it makes me a little confusing because usually I only focus on the 3 suggested words and now I also have to look at the field where I write the words. It leads to more eye movement which makes the whole typing process slower.
Maybe this could be an optional feature setting.

**Comment by dslul** [comment]
I agree that this should be an optional feature, because those who are accustomed to aosp (like me) may prefer this behaviour.

**Comment by kojid0** [comment]
tbh this is one of the main reasons I don't use openboard as a daily driver. Typing process has to be fast&simple but having to re-read words at another place of the screen (keyboard word suggestion vs text field) makes it slow.

**Comment by kojid0** [comment]
Bounty: 10€ 😊

**Comment by stefrey** [comment]
I'd be very happy too to have that feature! Since I have to press the spacebar in most of the cases after having written a word, it is more efficient to press the displayed word in the suggestion bar, because this ads a space too. This way, the user works more in the suggestion bar instead of switching around between space and suggestion bar. So it would be great to have a setting to enable that the word currently written is 1:1 displayed in the center of the suggestion.
